### ISSUE TITLE
Feature request: Add CompletionStages helper as an alias with akka.dispatch.Futures

### ISSUE THREAD

**Issue by hepin1989** [akka/akka#26111]
Currently, Akka has `akka.dispatch.Futures` for Java API but none for `CompletionStage`, even there is `CompletableFuture#allOf` and `CompletableFuture#anyOf`,but there are no more rich operators.

eg:
![image](https://user-images.githubusercontent.com/501740/49871423-30d6f500-fe51-11e8-9028-b9a8c549b6ab.png)

I opened this issue up because today I saw one of my workmate at HemaFresh is using `akka.dispatch.Futures` from Java. For what we have in our TQL implementation,we do have so many helpers to work with `CompletableStage`.
![image](https://user-images.githubusercontent.com/501740/49871545-80b5bc00-fe51-11e8-842b-3cd025381a81.png)

eg:
```java
  /**
   * 根据一个 {@link CompletableFuture} 的 {@link Iterable} ,并应用一个折叠函数，从而异步产生一个值。
   * 需要注意的是，这里对函数的应用是同步执行的。
   *
   * @param futures 需要进行异步折叠的所有 {@link CompletableFuture}.
   * @param zero    初始值；
   * @param acc     用于将值合并到一起的函数;
   * @return 返回一个 {@link CompletableFuture}，其中包含了折叠后的值。
   */
  public static <T, U> CompletableFuture<U> fold(
    final Iterable<? extends CompletableFuture<? extends T>> futures,
    final U zero,
    final BiFunction<? super U, ? super T, ? extends U> acc) 
```

I think Akka could provide something like this to helper Java developer work with `CompletionStage` more directly but not with the `akka.dispatch.Futures`

**Comment by hepin1989** [comment]
Yes, I hope there will be a counterpart of `akka.dispatch.Futures` for `CompletionStage`.

**Comment by patriknw** [comment]
I'm also skeptical to adding various general purpose utilities to Akka just because they are missing in the standard library. We would probably not do that for `Futures` either if it wasn't that the Scala Future started in Akka before it was incorporated into the Scala standard library.

**Comment by He-Pin** [comment]
![image](https://user-images.githubusercontent.com/501740/169382264-9468614c-770e-4178-9526-f1bb2b32c652.png)

**Comment by Captain1653** [comment]
This issue is quite old and has not been active for 1 year.

Should this issue be closed?

**Comment by He-Pin** [comment]
Yes, and feel free to reopen someday., and now jdk add more help functions now.
